### ISSUE TITLE
cpu.Percent always reports 0% used on a Debian server.

### ISSUE THREAD

**Issue by ecsumed** [shirou/gopsutil#753]
**Describe the bug**
Even with significant load on the system, `cpu.Percent` reports 0% used.

**To Reproduce**
1. ```stress -c 3 &``` on a 1 core Debian 8 server.

2. Run main.go
```go
// main.go
package main

import (
	"fmt"
	"github.com/shirou/gopsutil/cpu"
)

func main() {
	if val, err := cpu.Percent(1, true); err != nil {
		fmt.Errorf("Failed to get cpu percentage: %s", err)
		fmt.Println(val)
	} else {
		fmt.Println(val)
	}
}
```
**Output**
```
$ w ; go run main.go ; w
 09:33:52 up 87 days, 22:37,  2 users,  load average: 2.26, 2.06, 2.71
...
go: finding github.com/shirou/gopsutil/cpu latest
[0]
 09:33:54 up 87 days, 22:38,  2 users,  load average: 2.40, 2.10, 2.72
...
```

**Expected behavior**
`cpu.Percent` should report 100% usage.

**Environment**
```
$ cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 8 (jessie)"
NAME="Debian GNU/Linux"
VERSION_ID="8"
VERSION="8 (jessie)"
ID=debian
HOME_URL="http://www.debian.org/"
SUPPORT_URL="http://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"

$ uname -a
Linux 278624.cloudwaysapps.com 3.16.0-7-amd64 #1 SMP Debian 3.16.59-1 (2018-10-03) x86_64 GNU/Linux
```

**Comment by ecsumed** [comment]
Apologies. False alert. I should've looked a bit harder.

After some more digging, I found this https://github.com/shirou/gopsutil/issues/568#issuecomment-446009859 . Ran the script and it worked.

I'm actually porting a python script to go and in the python version, the interval is just an int for seconds. So if a 1 int interval is passed here, it's actually stands for 1 _nanosecond_ , while in the Python version a 1 stands for a _second_ . 

The nanosecond is too short to report any significant usage and thus the 0% usage.
